src/screens/navscreens/PickupScreen.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
Can you name this file PickupDetailsScreen and instead of putting it in the MainDrawer stack, can you place it in the main stack? That way, when you hit the back button on android, it will go back to the 'MyPickupScreen' page, not the Home page.